test/JDBC/input/BABEL-3101.sql

```diff
@@ -41,7 +41,7 @@ BEGIN
 END
 GO
 
-SELECT ISOWeek_3101(GETDATE())
+SELECT ISOWeek_3101(CAST('25 July 2022 01:23:45' AS datetime))
```

Comment:
this was a negative test where we wanted the function to fail, so please add a variation which fails